### PR TITLE
Nav

### PR DESCRIPTION
**Pull request by JM066**

### CODE REVIEW

File: src/components/Layout/Navbar/index.tsx [hunk 1]
```diff
@@ -1,9 +1,54 @@
...
+  const PAGES = [
+    {
+      id: "main",
+      name: "Main",
+      path: "/main",
+      show: true,
+    },
+    {
+      id: "smart-contract",
+      name: "Smart Contract",
```
Comment (BruceJi7) [inline src/components/Layout/Navbar/index.tsx:19]: Can that say Smart Contracts instead?

Comment (JM066) [reply src/components/Layout/Navbar/index.tsx:19]: Yes!

File: src/components/Layout/Navbar/index.tsx [hunk 2]
```diff
@@ -1,9 +1,54 @@
...
+import Link from "next/link";
+import classNames from "classnames";
+
+import ConunLogo from "@/assets/icons/conun-logo-head-bar.svg";
 
 import styles from "./Navbar.module.scss";
 
 function Navbar() {
-  return <div className={styles.Navbar}>NAV BAR</div>;
+  const PAGES = [
```
Comment (BruceJi7) [inline src/components/Layout/Navbar/index.tsx:10]: What if we add this const to a const file instead?

Comment (JM066) [reply src/components/Layout/Navbar/index.tsx:10]: Yes! It is done!

File: src/components/Layout/Navbar/index.tsx [hunk 3]
```diff
@@ -1,9 +1,54 @@
...
+            <a>
+              <ConunLogo />
+            </a>
+          </Link>
+        </div>
+        <div className={styles.Navs}>
+          {PAGES.map((nav) => {
+            return (
+              <Link key={nav.id} href={nav.path}>
+                <a className={classNames(styles.Nav)}>{nav.name}</a>
```
Comment (ivanms1) [inline src/components/Layout/Navbar/index.tsx:44]: `classNames` not needed

File: src/components/Layout/Footer/index.tsx [hunk 4]
```diff
@@ -1,9 +1,22 @@
...
+import { PAGES_FOOTER } from "../../../const";
 
 import styles from "./Footer.module.scss";
 
 function Footer() {
-  return <div className={styles.Footer}>FOOTER</div>;
+  return (
+    <div className={styles.Footer}>
+      {PAGES_FOOTER.map((logo) => {
+        return (
```
Comment (BruceJi7) [inline src/components/Layout/Footer/index.tsx:12]: Might be able to get rid of the return there

**Review by BruceJi7 — APPROVED** [review]

**Pull request merged**